lib/credit_card_validations.rb

```diff
@@ -1,4 +1,5 @@
 require 'credit_card_validations/version'
+require 'credit_card_validations/error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.